sources/platform/actors/development/actor_definition/source_code/example_setup.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Example setup
+description: See an example setup of the Actor's Dockerfile and source code.
+slug: /actors/development/actor-definition/source-code/example-setup
+sidebar_position: 1
+---
+
+**See an example setup of Actor's Dockerfile and source code.**
+
+---
+
+Let's take a look at the example JavaScript Actor's source code. The following Dockerfile
+
+```dockerfile
+FROM apify/actor-node:16
+
+COPY package*.json ./
+
+RUN npm --quiet set progress=false \
+    && npm install --omit=dev --omit=optional \
+    && echo "Installed NPM packages:" \
+    && (npm list --omit=dev --all || true) \
+    && echo "Node.js version:" \
+    && node --version \
+    && echo "NPM version:" \
+    && npm --version \
+    && rm -r ~/.npm
+
+COPY . ./
+
+CMD npm start --silent
+```
+
+will build the Actor from the `apify/actor-node:16` image, copy the `package.json` and `package-lock.json` files to the `/home/node` directory.
```

Comment:
Did you double check that it really copies to that directory? Maybe it's different now. Also, do we really need a guide on how Docker works here?

Reply:
I removed that completely. It's redundant information.